Source/WebCore/platform/mediastream/mac/ScreenCaptureKitSharingSessionManager.mm

```diff
@@ -542,11 +542,13 @@ - (void)stopObservingPicker:(SCContentSharingPicker *)picker
 
     auto activeSource = m_activeSources[index];
     m_activeSources.remove(index);
-
     if (!activeSource)
         return;
 
-    auto sharingSession = activeSource->sharingSession();
+    if (!promptingInProcess())
+        cancelPicking();
```

Comment:
Should we do this check even if there is no activeSource?

Reply:
Looks like we should. If there are no active sources, we do even more teardown in cancelPicking().

Reply:
Good idea, fixed